### PR TITLE
Adding documentation for experimentalModifyObstructiveThirdPartyCode

### PR DESCRIPTION
**Pull request by mschile**

### CODE REVIEW

File: content/guides/references/experiments.md [hunk 1]
```diff
@@ -21,12 +21,13 @@ options below to enable or disable experiments. See our
...
+| `experimentalSessionAndOrigin`                | `false` | Enables cross-origin and improved session support, including the [`cy.origin()`](/api/commands/origin) and [`cy.session()`](/api/commands/session) commands. Only available in end-to-end testing.                                                                                                                             |
+| `experimentalSourceRewriting`                 | `false` | Enables AST-based JS/HTML rewriting. This may fix issues caused by the existing regex-based JS/HTML replacement algorithm. See [#5273](https://github.com/cypress-io/cypress/issues/5273) for details.                                                                                                                         |
+| `experimentalModifyObstructiveThirdPartyCode` | `false` | Whether Cypress will search for and replace obstructive code in third party .js or .html files. NOTE: Setting this flag removes Subresource Integrity (SRI).                                                                                                                                                                   |
```
Comment (AtofStryker) [inline content/guides/references/experiments.md:30]:
```suggestion
| `experimentalModifyObstructiveThirdPartyCode` | `false` | Whether Cypress will search for and replace obstructive code in third party .js or .html files. NOTE: Setting this flag removes [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity).                                                                                                                                                                   |
```

**Review by AtofStryker — APPROVED** [review]

File: content/api/commands/origin.md [hunk 2]
```diff
@@ -43,6 +43,21 @@ to visit a page in your application.
...
+<strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
+Obstructive Third Party Code</strong>
+
+By default Cypress will search through the response streams coming from your
+server on first party `.html` and `.js` files and replace code that matches
+patterns commonly found in framebusting. When using the `cy.origin()` command,
+the third party code may also need to be modified for framebusting techniques.
+This can be enabled by setting
+the [`experimentalModifyObstructiveThirdPartyCode`](/guides/references/experiments) flag
+to `true` in the Cypress config.
```
Comment (emilyrohrbough) [inline content/api/commands/origin.md:57]:
```suggestion
to `true` in the Cypress configuration.
```

File: content/guides/references/experiments.md (hunk 1 above)
Comment (emilyrohrbough) [reply content/guides/references/experiments.md:30]: do we need backticks around .js and .html?

File: content/guides/references/experiments.md [hunk 3]
```diff
@@ -21,12 +21,13 @@ options below to enable or disable experiments. See our
...
 
 ## History
```
Comment (emilyrohrbough) [inline content/guides/references/experiments.md:32]: can you update the page history?

File: content/guides/references/experiments.md (hunk 1 above)
Comment (emilyrohrbough) [reply content/guides/references/experiments.md:30]: is this available in both e2e & ct testing?

File: content/guides/references/experiments.md (hunk 3 above)
Comment (mschile) [reply content/guides/references/experiments.md:32]: updated [61c7db9](https://github.com/cypress-io/cypress-documentation/pull/4622/commits/61c7db9b4663fb1323c5b34b6a6359db88538684)

File: content/api/commands/origin.md (hunk 2 above)
Comment (mschile) [reply content/api/commands/origin.md]: updated [61c7db9](https://github.com/cypress-io/cypress-documentation/pull/4622/commits/61c7db9b4663fb1323c5b34b6a6359db88538684)

**Review by emilyrohrbough — APPROVED** [review]

File: content/guides/references/experiments.md (hunk 1 above)
Comment (mschile) [reply content/guides/references/experiments.md]: updated [61c7db9](https://github.com/cypress-io/cypress-documentation/pull/4622/commits/61c7db9b4663fb1323c5b34b6a6359db88538684)

Comment (mschile) [reply content/guides/references/experiments.md]: > is this available in both e2e & ct testing?

Yes, I think it's okay to be available for both testing types.

**Review by debrisapron — APPROVED** [review]
🚀

**Pull request merged**